src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -571,10 +571,15 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
         return enrollmentDetails;
     }
 
-    /* All empty lines or lines with only white spaces will be skipped.
-     * The invalidity info is in HTML format.
+    /**
+     * This method builds the studentList from user input lines and check the validity of each student instance created.
+     *
+     * @param lines  the enrollment lines entered by the instructor.All empty lines or lines with only white
```

Comment:
The second sentence is about the method behavior, not about the parameter. It should be moved up.